Automattic/jetpack
# Add Termination Dialog Survey

#### Changes proposed in this Pull Request:

* Add a new component, `multiple-choice-question`
* Add front-end support for the endpoint added in #
* Enhance the Disconnect Dialog from #13517 to now have a survey step that mirrors [the survey on the Calypso side](https://github.com/Automattic/wp-calypso/blob/70bbfdf7dae43b78fd8295f9a97881386c7352b1/client/my-sites/site-settings/disconnect-site/disconnect-survey.jsx)
* Minor fix to `FormTextInput` Component to avoid throwing console errors


##### Before

<img width="1002" alt="Screen Shot 2019-11-08 at 4 47 26 PM" src="https://user-images.githubusercontent.com/2810519/68519723-b6bd1500-0247-11ea-8471-b8f74ad660cb.png">

##### After

<img width="450" alt="Screen Shot 2019-11-08 at 4 45 00 PM" src="https://user-images.githubusercontent.com/2810519/68519706-94c39280-0247-11ea-9e00-09415353ab0f.png">
<img width="450" alt="Screen Shot 2019-11-08 at 4 45 04 PM" src="https://user-images.githubusercontent.com/2810519/68519710-a0af5480-0247-11ea-9723-65125440b06b.png">


#### Is this a new feature or does it add/remove features to an existing part of Jetpack?

See parBdM-og-p2

#### Testing instructions:






1. Navigate to `/wp-admin/admin.php?page=jetpack#/dashboard`
2. Click "Manage Site Connection"
<img width="403" alt="Screen Shot 2019-09-24 at 3 53 46 PM" src="https://user-images.githubusercontent.com/2810519/65556153-84e42f00-dee3-11e9-98b4-cf02e6c9c7b0.png">
3. Verify that your dialog roughly matches the first step of the above "After" screenshot.
4. Click continue and verify the dialog roughly matches the second step of the above "After" screenshot
5. Chose an option on the survey and click "Disconnect"
6. Navigate to D32171-code and follow steps 4 & 5 from section v2 API to confirm survey submission

#### Proposed changelog entry for your changes:

* Add new survey step to Disconnect Dialog
